esy.json

```diff
@@ -5,11 +5,11 @@
   "license": "MIT",
   "version": "3.8.1",
   "dependencies": {
-    "@opam/dune": "2.9.3",
+    "@opam/dune": "3.6.2",
     "@opam/fix": "*",
     "@opam/menhir": " >= 20180523.0.0",
     "@opam/merlin-extend": " >= 0.6",
-    "@opam/ocamlfind": "*",
+    "@opam/ocamlfind": "1.9.5",
```

Comment:
Needed since ocamlfind 1.96 (latest) isn't on esy repo yet

Reply:
It currently is: https://github.com/esy-ocaml/esy-opam-override/tree/6/packages/ocamlfind.1.9.6 but esy-ocaml/esy-opam-override#130